samples/excel/worksheet/activeworksheet.yaml

```diff
@@ -0,0 +1,194 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Active worksheet
+description: Explore the APIs to get and set the active worksheet
+script:
+  content: |-
+    $("#setup").click(setup);
+
+    $("#get-active-worksheet").click(getActiveWorksheet);
+    $("#set-active-worksheet").click(setActiveWorksheet);
+
+    async function getActiveWorksheet() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+                sheet.load("name");
+                
+                await context.sync();
+
+                console.log(`The active worksheet is "${sheet.name}"`);
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+    async function setActiveWorksheet() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+                sheet.load("name");
+                
+                let sheets = context.workbook.worksheets;
+                sheets.load("items");
+
+                await context.sync();
+
+                let newSheet = sheet;
+
+                if (sheets.items.length > 1) {
+                    if (sheet.id === sheets.items[0].id) {
+                        newSheet = sheets.items[1];
+                    } else {
+                        newSheet = sheets.items[0];
+                    }
+                    newSheet.load("name");
+
+                    newSheet.activate();
+
+                    await context.sync();
+                } else {
+                    console.log("Cannot change the active worksheet since there is only one sheet in the workbook");
+                }
+
+                console.log(`The active worksheet is "${newSheet.name}"`);
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+    async function setup() {
+        try {
+            await Excel.run(async (context) => {
+
+                let sheets = context.workbook.worksheets;
+                sheets.load("items");
+                
+                await context.sync();
+
+                if (sheets.items.length < 2) {
```

Comment:
Out of curiosity, why add one?  or of you so want to add one, maybe call the setup button "add sheet"